index.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: blue;
+}
```

Comment:
Well done! 💯 
Thank you for making sure your .css file includes one class with styles for your header as required by the project. 🙏🏾